### PR TITLE
add install instructions for rpm and tar

### PR DESCRIPTION
**Pull request by samccann**

### CODE REVIEW

File: downstream/modules/navigator/proc-installing-navigator-rhel-rpm.adoc [hunk 1]
```diff
@@ -13,16 +13,22 @@ You can install {Navigator} on Red Hat Enterprise Linux (RHEL) from an RPM.
...
+* You registered your system with Red Hat Subscription Manager.
 
 
 .Procedure
 
-. Download the RPM.
+. Enable the repository for RHEL 8.
++
+----
+$ sudo subscription-manager repos --enable ansible-navigator-1.0.0.el8ap.noarch.rpm
```
Comment (simaishi) [inline downstream/modules/navigator/proc-installing-navigator-rhel-rpm.adoc:24]: The repo name will be: `automation-platform-2.0-for-rhel-8-x86_64-rpms`

Comment (samccann) [reply downstream/modules/navigator/proc-installing-navigator-rhel-rpm.adoc]: fixed, thanks!

**Comment by samccann** [comment]
Spredzy internally stated that the commands for the RPM install are correct.

**Comment by samccann** [comment]
Added updates to tar procedure based on slack discussion.

**Comment by cbudz** [comment]
lgtm 👍

**Pull request merged**